homeassistant/components/google_assistant/http.py

```diff
@@ -219,6 +236,56 @@ async def async_report_state(self, message, agent_user_id: str):
         }
         await self.async_call_homegraph_api(REPORT_STATE_BASE_URL, data)
 
+    @callback
+    def async_enable_local_sdk(self):
+        """Enable the local SDK."""
+        for key, value in self._store.agent_user_ids.items():
+            try:
+                webhook.async_register(
+                    self.hass,
+                    DOMAIN,
+                    "Local Support for " + key,
+                    value[STORE_GOOGLE_LOCAL_WEBHOOK_ID],
+                    self._handle_local_webhook,
+                )
+            except ValueError:
+                _LOGGER.info("Webhook handler is already defined!")
+                return
```

Comment:
Returning here causing others to not be set up.

Reply:
Now explicitely stating that the setup has failed and falling back to cloud fulfillment, where this return was there intentionally to serve that same purpose. Also cleaning up unused webhooks now.